batch.md

```diff
@@ -119,21 +157,29 @@ A builder API is provided to construct an instance of `FlinkPravegaOutputFormat`
 |`withEventRouter`|The router function which determines the routing key for a given event.|
 
 ### Output Stream
-Each stream in Pravega is contained by a scope.  A scope acts as a namespace for one or more streams. The builder API accepts both _qualified_ and _unqualified_ stream names.  By qualified, we mean that the scope is explicitly specified, e.g. `my-scope/my-stream`.   Unqualified stream names are assumed to refer to the default scope as set in the `PravegaConfig`.
+
+Each stream in Pravega is contained by a scope.  A scope acts as a namespace for one or more streams.  The `FlinkPravegaReader` is able to read from numerous streams in parallel, even across scopes.  The builder API accepts both **qualified** and **unqualified** stream names.  

```

Comment:
This line is not correct. May be a copy & paste proble? When writing to Pravega, you cannot pass more than one stream.
```
The `FlinkPravegaReader` is able to read from numerous streams in parallel, even across scopes.
```

Reply:
Accepted suggestion, edited please check